app/models/chatroom.rb

```diff
@@ -1,6 +1,12 @@
 class Chatroom < ApplicationRecord
   # TODO: You should validate that there are exactly two users in a Chatroom
-
   has_many :messages, dependent: :destroy
   has_and_belongs_to_many :users
+  #validate :has_only_2_users, :on => :create
```

Comment:
You should read on how to perform custom ActiveRecord validations
https://guides.rubyonrails.org/active_record_validations.html#performing-custom-validations